vllm/entrypoints/llm.py

```diff
@@ -734,10 +740,29 @@ def _run_engine(
                                 f"est. speed input: {in_spd:.2f} toks/s, "
                                 f"output: {out_spd:.2f} toks/s")
                         pbar.update(1)
+            if self.llm_engine.cache_service_metrics is not None:
+                logger.info(
+                    "Cache service hit rate: by tokens: %.2f%%, by blocks: %.2f%%, total tokens hit %d, number of measurement collected %d",
+                    0 if self.llm_engine.cache_service_metrics.total_tokens == 0 else self.llm_engine.cache_service_metrics.hit_tokens/self.llm_engine.cache_service_metrics.total_tokens * 100,
+                    0 if self.llm_engine.cache_service_metrics.total_blocks == 0 else self.llm_engine.cache_service_metrics.hit_blocks/self.llm_engine.cache_service_metrics.total_blocks * 100, 
+                    self.llm_engine.cache_service_metrics.total_tokens, 
+                    self.llm_engine.cache_service_metrics.counter,  
+                )
+                logger.info(
+                    "Cache service time query avg %.4f std %.4f normalized mean %.4f std %.4f, time load avg %.4f std %.4f normalized mean %.4f std %.4f, time unload avg %.4f std %.4f normalized mean %.4f std %.4f, time update avg %.4f std %.4f normalized mean %.4f std %.4f, time to first token avg %.4f std %.4f, normalized time to first token %.4f std %.4f",
+                    np.mean(self.llm_engine.cache_service_metrics.time_query), np.std(self.llm_engine.cache_service_metrics.time_query), np.mean(self.llm_engine.cache_service_metrics.normalized_time_query), np.std(self.llm_engine.cache_service_metrics.normalized_time_query),
```

Comment:
em. for experiment, this is ok. I high suggest to use prometheus metrics so we can easily check histogram